scripts/src/models/orders.ts

```diff
@@ -240,9 +239,12 @@ export const Order = {
 	 * @param      {number}  limit
 	 * @return     {Promise<T[]>}  filtered orders including p2p
 	 */
-	async getAll<T extends Order>(filters: GetOrderFilters & ({ userId: string } | { walletAddress: string }), limit?: number): Promise<T[]> {
-		const contexts = await OrderContext.find(filters.userId ? { userId: filters.userId } : { wallet: filters.walletAddress }); // can be replaced by cache
-		const ids = contexts.map(c => c.orderId);
+	async getAll<T extends Order>(filters: GetOrderFilters & ({ userId: string } | { walletAddress: string }), limit: number = 25): Promise<T[]> {
+		const orderIdQuery = await (this.genericGet(filters)
+			.limit(limit + 1)
+			.getMany()); // +1 to account for p2p
```

Comment:
Move the comment to the right line, and maybe elaborate more, not sure I understand the reasoning.

Reply:
It was irrelevant that comment and the +1. I removed it and tested it in a unittest